src/SDKs/Network/Network.Tests/Tests/VirtualNetworkTests.cs

```diff
@@ -372,5 +372,105 @@ public void VirtualNetworkPeeringTest()
                 networkManagementClient.VirtualNetworks.Delete(resourceGroupName, vnet2Name);
             }
         }
+
+        [Fact]
+        public void VirtualNetworkUsageTest()
+        {
+            var handler1 = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+            var handler2 = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                var resourcesClient = ResourcesManagementTestUtilities.GetResourceManagementClientWithHandler(context, handler1);
```

Comment:
@number213 I don't see any changes to code that creates the actual Resource Manager client.
Could you point me to that change?